compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -76,4 +77,27 @@ bool isLoopPeelingEnabled(FunctionOpInterface funcOp) {
   return config && config.contains(kLoopPeelingAttrName);
 }
 
+/// Creates a string attribute containing the name of the attribute that is
+/// used to enable SCF lowerig for Topk operations.
+void setEnableTopkSCFLowerAttrName(IREE::LinalgExt::TopkOp topkOp) {
```

Comment:
Instead of adding a separate attribute name, `lowering_config` now has a dictionary attribute. You can set that as an entry to the dictionary attribute.

Reply:
Can you follow how we add the peeling attribute? I think you can implement something similar to above methods, and follow what has been done for peeling config. E.g.,

https://github.com/iree-org/iree/blob/6fdc30f1638aa8dc5c4af66b91233086376ec530/compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp#L1088-L1093